server_cmds.go

```diff
@@ -15,129 +15,149 @@ type ServerMethods struct {
 }
 
 // Instance represents basic information for a TeamSpeak 3 instance.
+// instanceinfo
 type Instance struct {
-	DatabaseVersion             int    `ms:"serverinstance_database_version"`
-	FileTransferPort            int    `ms:"serverinstance_filetransfer_port"`
-	MaxTotalDownloadBandwidth   uint64 `ms:"serverinstance_max_download_total_bandwidth"`
-	MaxTotalUploadBandwidth     uint64 `ms:"serverinstance_max_upload_total_bandwidth"`
-	GuestServerQueryGroup       int    `ms:"serverinstance_guest_serverquery_group"`
-	ServerQueryFloodCommands    int    `ms:"serverinstance_serverquery_flood_commands"`
-	ServerQueryFloodTime        int    `ms:"serverinstance_serverquery_flood_time"`
-	ServerQueryBanTime          int    `ms:"serverinstance_serverquery_ban_time"`
-	TemplateServerAdminGroup    int    `ms:"serverinstance_template_serveradmin_group"`
-	TemplateServerDefaultGroup  int    `ms:"serverinstance_template_serverdefault_group"`
-	TemplateChannelAdminGroup   int    `ms:"serverinstance_template_channeladmin_group"`
-	TemplateChannelDefaultGroup int    `ms:"serverinstance_template_channeldefault_group"`
-	PermissionsVersion          int    `ms:"serverinstance_permissions_version"`
-	PendingConnectionsPerIP     int    `ms:"serverinstance_pending_connections_per_ip"`
+	DatabaseVersion                int    `ms:"serverinstance_database_version"`
+	FileTransferPort               int    `ms:"serverinstance_filetransfer_port"`
+	MaxTotalDownloadBandwidth      uint64 `ms:"serverinstance_max_download_total_bandwidth"`
+	MaxTotalUploadBandwidth        uint64 `ms:"serverinstance_max_upload_total_bandwidth"`
+	GuestServerQueryGroup          int    `ms:"serverinstance_guest_serverquery_group"`
+	ServerQueryFloodCommands       int    `ms:"serverinstance_serverquery_flood_commands"`
+	ServerQueryFloodTime           int    `ms:"serverinstance_serverquery_flood_time"`
+	ServerQueryBanTime             int    `ms:"serverinstance_serverquery_ban_time"`
+	TemplateServerAdminGroup       int    `ms:"serverinstance_template_serveradmin_group"`
+	TemplateServerDefaultGroup     int    `ms:"serverinstance_template_serverdefault_group"`
+	TemplateChannelAdminGroup      int    `ms:"serverinstance_template_channeladmin_group"`
+	TemplateChannelDefaultGroup    int    `ms:"serverinstance_template_channeldefault_group"`
+	PermissionsVersion             int    `ms:"serverinstance_permissions_version"`
+	PendingConnectionsPerIP        int    `ms:"serverinstance_pending_connections_per_ip"`
+	ServerQueryMaxConnectionsPerIP int    `ms:"serverinstance_serverquery_max_connections_per_ip"`
 }
 
 // ServerConnectionInfo represents the connection info for a TeamSpeak 3 instance.
+// hostinfo
```

Comment:
Not sure why this was added?

Reply:
same as above.
I could remove them if you think that they are redundant.